ajax/libs/mavo/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "mavo",
+  "version": "0.2.0",
+  "description": "Create web applications with HTML and CSS",
+  "main": "mavo.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mavoweb/mavo.git"
+  },
+  "author": "Lea Verou",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/mavoweb/mavo/issues"
+  },
+  "homepage": "https://mavo.io",
+  "autoupdate": {
+    "npmName": "mavo",
+    "npmFileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

```suggestion
          "*.@(js|css)"
```

Looking at the files in the npm dist directory, it looks like you'll only want the JS/CSS files at the root and nothing else?

[Glob test](https://www.digitalocean.com/community/tools/glob?glob=%2A.%40%28js%7Ccss%29&tests=maps%2Fmavo-nodeps.js.map&tests=maps%2Fmavo.es5.min.js.map&tests=maps%2Fmavo.css.map&tests=maps%2Fmavo.min.js.map&tests=maps%2Fmavo.es5.js.map&tests=maps%2Fmavo.js.map&tests=mavo-nodeps.js&tests=0.1.4%2Fmaps%2Fmavo.es5.min.js.map&tests=0.1.4%2Fmaps%2Fmavo.css.map&tests=0.1.4%2Fmaps%2Fmavo.min.js.map&tests=0.1.4%2Fmaps%2Fmavo.es5.js.map&tests=0.1.4%2Fmaps%2Fmavo.js.map&tests=0.1.4%2F.DS_Store&tests=0.1.4%2Fmavo.css&tests=0.1.4%2Fmavo.es5.js&tests=0.1.4%2Fmavo.js&tests=0.1.4%2Fmavo.min.js&tests=0.1.4%2Fmavo.es5.min.js&tests=index.html&tests=.DS_Store&tests=mavo.css&tests=deps.js&tests=mavo.es5.js&tests=mavo.js&tests=mavo.min.js&tests=mavo.es5.min.js)